conf/local.conf.sample

```diff
@@ -94,7 +94,8 @@ USER_FEATURES += "~3g"
 USER_FEATURES += "~nfc"
 
 # When the meta-virtualization layer is enabled, enable the feature by default
-USER_FEATURES += "${@bb.utils.contains('BBFILE_COLLECTIONS', 'virtualization-layer', 'virtualization', '', d)}"
+USER_FEATURES_append_mel = "${@bb.utils.contains('BBFILE_COLLECTIONS', 'virtualization-layer', ' virtualization', '', d)}"
```

Comment:
@Noor-Ahsan It looks like the only place that this distro feature is used other than the meta-virtualization layer is in enabling kernel configs needed. Given the low impact, we might consider just unconditionally enabling the distro feature in mel.conf rather than having yet another change controlled by layer inclusion. Thoughts?

Reply:
@kergoth, how would the user be able to disable the feature then? Commenting out image features?

Reply:
Yes, that would prevent inclusion of anything from meta-virtualization. If they really don't want the kernel config, you can still do USER_FEATURES += "~virtualization" to undo what mel.conf adds to DISTRO_FEATURES.